src/Tools/Shared/CommandLine/CommandLineApplicationExtensions.cs

```diff
@@ -21,6 +21,17 @@ public static void OnExecute(this CommandLineApplication app, Action action)
                     return 0;
                 });
 
+        public static CommandOption Option(this CommandLineApplication command, string template, string description)
```

Comment:
Is this needed? I don't see it used anywhere. Also seems a little too magical for a general use-case extension

Reply:
Yes it's needed; none of the `Option(...)` calls in these tools pass a `CommandOptionType`.